doc/linting.md

```diff
@@ -0,0 +1,23 @@
+# Linging Page
+
+The extension provides linting which looks like in the following screenshot:
+
+[![Linting](../images/linting/code.jpg)]()
+
+Also it populates the Problems panel.
+
+For the code:
+
+```rust
+fn foo(i: i32) {}
+
+fn main() {
+    foo(2i64);
+}
+```
+
+The Problems panel looks like in the following screenshot:

```

Comment:
"Would" because the code above is a hypothetical example/scenario. You can use "will" if you prefer. Either works.